views/forms/ChartForm.jsx

```diff
@@ -10,14 +10,19 @@ import BarChartFields from './BarChartFields'
 import { chartStyles } from '../styles/chart_styles'
 import { dark_sky_blue } from '../constants/styles'
 
-const ChartForm = ({ classes, handleSubmit }) => {
+const ChartForm = ({ classes, handleSubmit, user }) => {
   const [formValues, setFormValues] = useState({
     title: '',
     description: '',
     assessment: '',
     variable: '',
     fieldLabelValueMap: [
-      { value: '', label: '', color: dark_sky_blue }
+      { 
+        value: '', 
+        label: '', 
+        color: dark_sky_blue,
+        targetValues: user.userAccess.map((site) => ({ site, value: '' })) || []
```

Comment:
couple questions here:
when will we hit the right hand side of the `||` -- could we just rely on `map` to return an empty `[]`?
same question re: modifying `userAccess`

Reply:
Thanks. The backend is already sending an empty array if no access exists. Could use your help understanding the 'adding things to something called `userAccess`' part.